sonic-xcvrd/scripts/xcvrd

```diff
@@ -0,0 +1,430 @@
+#!/usr/bin/env python
+
+"""
+    Xcvrd
+    Transceiver information update daemon for SONiC
+"""
+
+try:
+    import getopt
+    import os
+    import imp
+    import signal
+    import subprocess
+    import sys
+    import syslog
+    import time
+    import threading
+    from swsscommon import swsscommon
+except ImportError, e:
+    raise ImportError (str(e) + " - required module not found")
+
+#============================= Constants =============================
+
+VERSION = '1.0'
+
+SYSLOG_IDENTIFIER = "xcvrd"
+XCVRD_MODULE_NAME = "xcvrd"
+
+PLATFORM_ROOT_PATH = '/usr/share/sonic/device'
+SONIC_CFGGEN_PATH = '/usr/local/bin/sonic-cfggen'
+MINIGRAPH_PATH = '/etc/sonic/minigraph.xml'
+HWSKU_KEY = 'DEVICE_METADATA.localhost.hwsku'
+PLATFORM_KEY = 'DEVICE_METADATA.localhost.platform'
+
+PLATFORM_SPECIFIC_MODULE_NAME = "sfputil"
+PLATFORM_SPECIFIC_CLASS_NAME = "SfpUtil"
+
+# Global platform-specific sfputil class instance
+platform_sfputil = None
+
+# platform directory in base image
+PLATFORM_ROOT = "/usr/share/sonic/device"
+
+# platform root directory inside docker
+PLATFORM_ROOT_DOCKER = "/usr/share/sonic/platform"
+
+REDIS_HOSTNAME = "localhost"
+REDIS_PORT = 6379
+REDIS_TIMEOUT_USECS = 0
+SELECT_TIMEOUT = 1000
+
+DOM_INFO_UPDATE_PERIOD = 60
+
+SFP_STATUS_PLUGIN = '1'
```

Comment:
>SFP_STATUS_PLUGIN [](start = 0, length = 17)

plugin->inserted
plugout->removed

Reply:
Suggest rename, plug* are not good names.

---
In reply to: [200794543](https://github.com/Azure/sonic-platform-daemons/pull/17#discussion_r200794543) [](ancestors = 200794543)